packages/session/README.md

```diff
@@ -0,0 +1,36 @@
+# `@prairielearn/session`
+
+The implementation borrows heavily from prior art such as [`express-session`](https://github.com/expressjs/session) and [`fastify-session`](https://github.com/fastify/session). However, the semantics and functionality have been changed to better suit PrairieLearn's needs.
```

Comment:
As long as we're justifying the package, maybe explain here why we don't just use those? What are those needs?

Reply:
I can add the following text:

> Specifically:
> - We need to have more precise control over when the session is written back to the session store. `express-session` will try to write the session on every request, which produces an undesirable amount of load on the database.
> - We need to have more precise control over when new/updated cookies are sent back to the client. In the near future, we'll need to avoid writing these cookies on subdomains.

Does that look good? Would you want any more information?

Reply:
lgtm

Reply:
Added in a8c3864.